Parser/string_parser.c

```diff
@@ -491,7 +502,8 @@ fstring_find_literal(Parser *p, const char **str, const char *end, int raw,
             *literal = PyUnicode_DecodeUTF8Stateful(literal_start,
                                                     s - literal_start,
                                                     NULL, NULL);
-        } else {
+        }
+        else {
```

Comment:
unintentional extra line break added on the else?

Reply:
That's the "PEP 7 nit" mentioned in the description. The "else {" should be on its own line. This is the only incorrect instance I saw, and since I was working on that code in that function, I thought I'd fix it.